include/wx/qt/private/treeitemfactory.h

```diff
@@ -0,0 +1,123 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/treeitemfactory.h
+// Purpose:     Factory to create text edit controls for the tree items
+// Author:      Graham Dawes
+// Modified by:
+// Created:     02/2019
```

Comment:
Let's standardize on the One True Date Format:
```suggestion
// Created:     2019-02-07
```

Reply:
ISO-8601 all the way!